src/main/java/com/ernie/HW6CustomStorages/MyHashSet.java

```diff
@@ -0,0 +1,41 @@
+package com.ernie.HW6CustomStorages;
+
+import java.util.HashMap;
+import java.util.Set;
+
+public class MyHashSet {
+    private final int BASE_CAPACITY = 8;
+    private HashMap<String, Boolean> baseHashMap;
```

Comment:
do not use HashMap, instead use array, ArrayList ot LinkedList

Reply:
changed to array in [https://github.com/my20yeezy/Learning-at-Andersen/pull/2/commits/dd8609aa770a8378a94796227eba31c14e975425](url)